modules/datadog-integration/component.yaml

```diff
@@ -0,0 +1,38 @@
+# 'datadog-integration' component vendoring config
```

Comment:
should we include `component.yaml` in upstreams?

Reply:
we havent in the past, but it could be helpful

Reply:
Yup, just removed it